tabuckner/broskichat
# User auth

### Project Changed


**- What I Did**
- Created chatUser model
- Loopback hashed/salts passwords
- Verified data posted to MongoAtlas through API Explorer

**- How I Did It**
- Added chat-user files in models folder
- Adjusted model-config file

**- How to Test/Verify It**

- Start server
- Through API explorer "try it out" under GET /chatUsers
- Should see TestUser w/email & id

If you want to add a test user:
- Create new user under POST /chatUsers
	* Paste in data box: `{"username": "testUsername", "email": "testEmail", "password":"testPassword"}`

- Should see TestUser as well as your user




**- An Image to Make Us Giggle (not mandatory but encouraged)**

Current feels:

![](https://static.fjcdn.com/gifs/Kungfu_469d1a_1059134.gif)
